resources/scripts/components/server/features/JavaVersionModalFeature.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { ServerContext } from '@/state/server';
+import Modal from '@/components/elements/Modal';
+import tw from 'twin.macro';
+import Button from '@/components/elements/Button';
+import setSelectedDockerImage from '@/api/server/setSelectedDockerImage';
+import FlashMessageRender from '@/components/FlashMessageRender';
+import useFlash from '@/plugins/useFlash';
+import { SocketEvent, SocketRequest } from '@/components/server/events';
+import Select from '@/components/elements/Select';
+
+const JavaVersionModalFeature = () => {
+    const [ visible, setVisible ] = useState(false);
+    const [ loading, setLoading ] = useState(false);
+
+    const uuid = ServerContext.useStoreState(state => state.server.data!.uuid);
+    const status = ServerContext.useStoreState(state => state.status.value);
+    const { clearFlashes, clearAndAddHttpError } = useFlash();
+    const { connected, instance } = ServerContext.useStoreState(state => state.socket);
+    let selectedVersion = 'ghcr.io/pterodactyl/yolks:java_16';
+
+    const dockerImageList = [
+        { name: 'Java 8', image: 'ghcr.io/pterodactyl/yolks:java_8' },
+        { name: 'Java 11', image: 'ghcr.io/pterodactyl/yolks:java_11' },
+        { name: 'Java 16', image: 'ghcr.io/pterodactyl/yolks:java_16' },
+    ];
+
+    useEffect(() => {
+        if (!connected || !instance || status === 'running') return;
+
+        const errors = [ 'minecraft 1.17 requires running the server with java 16 or above',
+            'java.lang.unsupportedclassversionerror',
+            'unsupported major.minor version',
+            'has been compiled by a more recent version of the java runtime' ];
```

Comment:
Put each error on it's own line, not on the same line as the array open and close brackets.

Reply:
Fixed in next commit